src/coreclr/tests/helixpublishwitharcade.proj

```diff
@@ -82,6 +82,7 @@
     <HelixRuntimeRid Condition="'$(TargetOSSpec)' == 'Linux'">linux-$(TargetArchitecture)</HelixRuntimeRid>
     <HelixRuntimeRid Condition="'$(TargetOSSpec)' == 'Linux_musl'">linux-musl-$(TargetArchitecture)</HelixRuntimeRid>
     <HelixRuntimeRid Condition="'$(TargetArchitecture)' == 'wasm'">browser-wasm</HelixRuntimeRid>
+    <HelixRuntimeRid Condition="'$(TargetOS)' == 'Android'">android-x64</HelixRuntimeRid>
```

Comment:
In the helix-queues-setup you seem to be checking for "android-x64" and "android-x86" - should this be somehow forked for the two architectures?

Reply:
According to https://github.com/dotnet/runtime/issues/33381, only x64 is needed. I will remove x86.